src/main/java/com/mathworks/ci/MatlabBuilderConstants.java

```diff
@@ -41,5 +41,10 @@
     static final String TEST_RUNNER_SCRIPT = "testScript = genscript(${PARAMS});\n" + "\n"
             + "disp('Running MATLAB script with content:');\n"
             + "disp(strtrim(testScript.writeToText()));\n"
-            + "fprintf('___________________________________\\n\\n');\n" + "run(testScript);\n" + "";
+            + "runner_file = testScript.writeToFile([tempname(tempname),'.m']);\n"
```

Comment:
Why is `tempname` being passed to `tempname`? If you're writing the script to a temp directory, can't the script just have a fixed, readable, name?

Reply:
yes, but since we are adding the temp in Path there is chance of readable static name collision in case of Matrix runs etc.

Reply:
Matrix runs don't run in the same MATLAB session though, right? Whatever you're adding to the path in one session is not going to effect what's going on in another session unless you savepath.

Reply:
its different sessions correct. But how do you think would it behave when 
fld1/runner.m and fld2/runner.m got added at same/ different time in path and one session tries to call runner.m ? will it still be tightly coupled to the session ? I prefer name being unique even if we have chance of name collision either by user or by our code itself even once in million.